rebar.config

```diff
@@ -14,13 +12,22 @@
             warn_unused_import,
             warnings_as_errors]}.
```

Comment:
Or, do you just mean right here in the `erl_opts` for the default profile, so that it is included when the library is locally compiled? Forgive my ignorance, but am I right to assume that this would not be respected when other applications include `jsn` as a dependency?